spec/navigator_spec.js

```diff
@@ -656,10 +656,10 @@ describe('Navigator', function () {
     it('invokes all actions with the request and options', function () {
       subject._invokeActions(request, options);
 
-      expect( target.actionOne ).toHaveBeenCalledWith(request, options);
-      expect( target.actionTwo ).toHaveBeenCalledWith(request, options);
-      expect( target.actionThree ).toHaveBeenCalledWith(request, options);
-      expect( target.actionFour ).toHaveBeenCalledWith(request, options);
+      expect( target.actionOne ).toHaveBeenCalledWith(request, options, jasmine.any(Object));
+      expect( target.actionTwo ).toHaveBeenCalledWith(request, options, jasmine.any(Object));
+      expect( target.actionThree ).toHaveBeenCalledWith(request, options, jasmine.any(Object));
+      expect( target.actionFour ).toHaveBeenCalledWith(request, options, jasmine.any(Object));
```

Comment:
Ha, no worries, I appreciate why it's there. Only alternative would be a custom constructor instance  or something but that's overkill.